router/build.go

```diff
@@ -54,6 +56,7 @@ func BuildHandlers(base *gin.RouterGroup) {
 			build.GET("", perm.MustRead(), api.GetBuild)
 			build.PUT("", perm.MustWrite(), middleware.Payload(), api.UpdateBuild)
 			build.DELETE("", perm.MustPlatformAdmin(), api.DeleteBuild)
+			build.DELETE("/cancel", executors.Establish(), perm.MustAdmin(), api.CancelBuild)
```

Comment:
Thoughts on updating this to `perm.MustWrite(` instead of `perm.MustAdmin()`?

Reply:
To start I figured admins was enough. Do you believe it should be MustWrite instead?

I'll merge this for now, but that's something we could come back to.